integreat_cms/cms/templatetags/content_filters.py

```diff
@@ -96,6 +96,28 @@ def remove(queryset, instance):
     return queryset.exclude(id=instance.id)
 
 
+@register.filter
+def sort_languages(other_languages, current_language):
+    """
+    This filter sorts languages in language tabs
+
+    :param current_language: The current language
+    :type current_language: string
```

Comment:
```suggestion
    :type current_language: ~integreat_cms.cms.models.languages.language.Language
```
A language is not a string.